### DISCUSSION TITLE
Using ColumnView in Relm4?

### DISCUSSION THREAD

**Discussion by cmdcolin** [Relm4/Relm4#448] [Q&A]
Hi there,
I was curious about using ColumnView in Relm4. I made a PR awhile back trying to use it in gtk4-rs (https://github.com/gtk-rs/gtk4-rs/pull/1111) but would be interested in doing it in Relm4 too!

**Comment by AaronErhardt** [comment]
Hi!

Sure, it'd be great to have an example for `ColumnView`. Currently, Relm4 does not add anything on top of the gtk4-rs APIs for `ColumnView`, but there is a pending PR to add an abstraction over `ListView`, which might be useful for `ColumnView` as well: https://github.com/Relm4/Relm4/pull/440

**Comment by cmdcolin** [comment]
looks like ColumnView support+example was added since this was posted, very cool! https://github.com/Relm4/Relm4/blob/main/examples/typed_column_view.rs
